src/API/users.js

```diff
@@ -0,0 +1,46 @@
+import { client } from '../utils/fetchClient';
+
+
+export const getUsers = async () => {
+  const response = await client.get('/users');
+
+  return response.data;
+};
+
+export const addUser = async (userData) => {
+  const response = await client.post('/users', {
+    name: userData.name,
+    username: userData.name,
```

Comment:
The `username` field is assigned the value of `userData.name`. If this is intentional, ensure that `name` and `username` are supposed to be the same. Otherwise, consider using `userData.username`.